cmd/restic/cmd_copy.go

```diff
@@ -203,10 +180,13 @@ func copyTree(ctx context.Context, srcRepo restic.Repository, dstRepo restic.Rep
 	packList := restic.NewIDSet()
 
 	enqueue := func(h restic.BlobHandle) {
-		pb := srcRepo.LookupBlob(h.Type, h.ID)
-		copyBlobs.Insert(h)
-		for _, p := range pb {
-			packList.Insert(p.PackID)
+		if !seenBlobs.Has(h.ID) {
```

Comment:
I don't think we need `seenBlobs` at all. Once `repository.Repack(Inner)` has run, all copied blobs will immediately also be returned by `dstRepo.LookupBlobSize`, even if the repository was not flushed yet.

Reply:
Well, that wasn't actually the case and will be fixed by https://github.com/restic/restic/pull/5614 .